### PR TITLE
SCA, CS, add more tests

### PR DESCRIPTION
**Pull request by SpacePossum**
just some dusting and such

### CODE REVIEW

File: tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php [hunk 1]
```diff
@@ -242,4 +242,34 @@ public function provide54Cases()
...
         );
     }
+
+    /**
+     * @dataProvider providePHP7Cases
+     * @requires PHP 7.0
+     */
+    public function testFixPHP7($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
```
Comment (keradus) [inline tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php]: `doTest`

**Comment by keradus** [comment]
:+1: but minor fix is needed

File: tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php (hunk 1 above)
Comment (SpacePossum) [inline tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php]: thanks! kind of an indicator how lang this has been in my stash :P

**Comment by keradus** [comment]
Travis is still red :(

**Comment by SpacePossum** [comment]
green now :)

**Comment by keradus** [comment]
awesome !

**Comment by GrahamCampbell** [comment]
:+1:

File: tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php [hunk 2]
```diff
@@ -268,6 +268,21 @@ public function providePHP7Cases()
     \RFb,
     \Fcc, \GFddZz
 {
+};',
+            ),
+                        array(
```
Comment (keradus) [inline tests/Fixer/ClassNotation/ClassDefinitionFixerTest.php]: btw, to big indent

**Comment by keradus** [comment]
please, squash and rebase

**Comment by SpacePossum** [comment]
done + pick

**Comment by keradus** [comment]
thanks !

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by SpacePossum** [comment]
picked in https://github.com/SpacePossum/PHP-CS-Fixer/pull/13 as well :)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**